Tests/Integration/src/ContainerConfiguratorTrait.php

```diff
@@ -36,6 +41,10 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
             if (self::VERSION_ID >= 40200) {
                 $loader->load($confDir . '/routing.yaml');
             }
+
+            if (class_exists(MessageBusInterface::class)) {
```

Comment:
Same thing here : `class_exists` used on an interface instead of `interface_exists`